internal/persistence/postgres/options.go

```diff
@@ -0,0 +1,111 @@
+package postgres
```

Comment:
Thanks for the refactor w/ copying what was done with the recent MySQL work. I didn't want to change the Postgres implementation with those changes, but figured we'd get to it eventually.